includes/html/modal/alert_rule_collection.inc.php

```diff
@@ -53,7 +55,9 @@
                             echo "
                                 <tr>
                                     <td>{$rule['name']}</td>
-                                    <td>{$rule['rule']}</td>
+                                    <td>";
+                            echo $rule['builder'] ? QueryBuilderParser::fromJson($rule['builder'])->toSql(false) : $rule['rule'];
```

Comment:
Should be `!empty($rule['builder'])` as builder key is likely to be not present on some rules.

Reply:
changed